homeassistant/components/ombi/__init__.py

```diff
@@ -57,9 +58,10 @@ def urlbase(value) -> str:
     {
         DOMAIN: vol.Schema(
             {
-                vol.Required(CONF_API_KEY): cv.string,
                 vol.Required(CONF_HOST): cv.string,
                 vol.Required(CONF_USERNAME): cv.string,
+                vol.Optional(CONF_API_KEY): cv.string,
+                vol.Optional(CONF_PASSWORD): cv.string,
```

Comment:
Make them exclusive with `vol.Exclusive`. Also add `cv.has_at_least_one_key` around the whole dict to make sure the user provides at least one of the options.